### PR TITLE
Update SE gain, add double-pe emission

### PR DESCRIPTION
**Pull request by JelleAalbers**
This adds a double photoelectron emission probability of 0.15 to fax. It is not applied for afterpulses (because they are not made by UV light, and because it is easier to code this way :-)

I also updated the single electron gain to 23 photons/single electron, see https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:aalbers:single_electron_gain_stability. We can probably derive a more accurate value later.

### CODE REVIEW

File: pax/config/XENON1T.ini [hunk 1]
```diff
@@ -244,6 +244,9 @@ s1_patterns_file =                    'XENON1T_s1_xyz_patterns_Feb2017.json.gz'
 s2_patterns_file =                    'XENON1T_s2_xy_patterns_top_updatedgeo.json.gz'
 s2_mean_area_fraction_top =           0.56             # See XeAnalysisScripts/mapmaking
 
+# Probability of oduble photoelectron emission
```
Comment (pdeperio) [inline pax/config/XENON1T.ini]: typo

Comment (JelleAalbers) [inline pax/config/XENON1T.ini]: Nah oduble photoelectron emission is a special magic process, don't you know ;-)

File: pax/simulation.py [hunk 2]
```diff
@@ -257,6 +257,13 @@ def make_pax_event(self):
 
             photon_detection_times = np.array(photon_detection_times)
 
+            # Add double photoelectron emission
+            if len(photon_detection_times):
+                n_dpe = np.random.binomial(len(photon_detection_times), p=self.config['p_double_pe_emision'])
+                if n_dpe:
+                    dpe_times = np.random.choice(photon_detection_times, n_dpe, replace=False)
+                    photon_detection_times = np.concatenate([photon_detection_times, dpe_times])
```
Comment (pdeperio) [inline pax/simulation.py]: Easy to add extra truth info for DPE?

Comment (JelleAalbers) [inline pax/simulation.py]: Not from this place in the code, though I agree it would be useful to store somewhere

**Pull request merged**